components/admin_console/billing/billing_summary.tsx

```diff
@@ -45,9 +53,210 @@ const noBillingHistory = (
 );
 
 const BillingSummary: React.FC = () => {
+    const subscription = useSelector((state: GlobalState) => state.entities.cloud.subscription);
+    const product = useSelector((state: GlobalState) => {
+        if (state.entities.cloud.products && subscription) {
+            return state.entities.cloud.products[subscription?.product_id];
+        }
+        return undefined;
+    });
+
+    let body = noBillingHistory;
+    if (subscription && subscription.last_invoice) {
+        const invoice = subscription.last_invoice;
+
+        const fullCharges = invoice.line_items.filter((item) => item.type === 'full');
+        const partialCharges = invoice.line_items.filter((item) => item.type === 'partial');
+
+        body = (
+            <div className='BillingSummary__lastInvoice'>
+                <div className='BillingSummary__lastInvoice-header'>
+                    <div className='BillingSummary__lastInvoice-headerTitle'>
+                        <FormattedMessage
+                            id='admin.billing.subscriptions.billing_summary.lastInvoice.title'
+                            defaultMessage='Last Invoice'
+                        />
+                    </div>
+                    {invoice.status === 'paid' &&
```

Comment:
There is another state that could appear "open" so maybe we want to have it into account

Reply:
That's news to me...Is that like a Pending state?

Reply:
Yep kind of. You can see [here](https://github.com/mattermost/customer-web-server/pull/257/files/f22143573bae995072b9cfb3cf07ea15dda7e99d#diff-e519a9713aedfa590f5a709c33f23d9d7a9dc9c20e95b0a1236870db46abff0aR124) is like "we don't know the state yet so we mark it as open"

Reply:
Ahh okay so it's more of a default case. I do the same thing on the Billing History screen so I'll mimic it here.